docs-parts/computation/01-autopopulate_lang4.rst

```diff
@@ -0,0 +1,28 @@
+The function ``parpopulate`` works identically to ``populate`` except that it uses a job reservation mechanism to allow multiple processes to populate the same table in parallel without collision.
+To enable ``parpopulate``, create the job reservation table ``<package>.Jobs`` which must have the following declaration:
```

Comment:
No, the job reservation table is created automatically.  You do not need to do anything to "enable" ``parpopulate``.
```suggestion
When running ``parpopulate`` for the first time, DataJoint will create a job reservation table and its class ``<package>.Jobs`` with the following declaration:
```